crates/runtime/src/http/routes.rs

```diff
@@ -75,6 +75,7 @@ pub(crate) fn routes(
             .route("/v1/search", post(v1::search::post))
             .route("/v1/tools", get(v1::tools::list))
             .route("/v1/tool/:name", post(v1::tools::post))
+            .route("/v1/eval/:name/run", post(v1::eval::post))
```

Comment:
```suggestion
            .route("/v1/evals/:name", post(v1::eval::post))
```

You don't need the `/run` a POST on the object is sufficient.  Or if this is truly a "eval run", then it should be `POST /v1/evals/:name/runs`

I also noticed that tools is incorrect. It should be plural, `/tools/`